firebase-vscode/src/stubs/open-stub.js

```diff
@@ -0,0 +1,5 @@
+const vscode = require("vscode");
+
+module.exports = async (target, options) => {
+  vscode.env.openExternal(target);
+};
```

Comment:
nit: should this be renamed to open-from-vscode ? open-stub seems to imply that its a stubbed out method